src/components/breadcrumb/README.md

```diff
@@ -13,3 +13,24 @@ Setting this to `display: none` will remove the associated slash.
   display: none;
 }
 ```
+
+
+
+Add `.bx--breadcrumb--no-trailing-slash` to `.bx--breadcrumb` to remove leading slashes. 
```

Comment:
Can you start this sentence with "Or you can" so that it reads like this:

> Or you can add `.bx--breadcrumb--no-trailing-slash` to `.bx--breadcrumb` to remove leading slashes.

Doing this small change makes it a little clearer that these are two different ways of removing a trailing slash.